src/server/service/roles/endpoints.js

```diff
@@ -0,0 +1,94 @@
+const { asyncMiddleware } = require('../../util/api');
+
+const db = require('./db');
+
+const { getUserById } = require('../../util/authenticationHelpers');
```

Comment:
Maybe we should have some more general helpers at some point, since I could see this being used in a lot of places besides authentication. Non-blocking for this though.